src/Hashids.php

```diff
@@ -79,7 +100,10 @@ class Hashids implements HashidsInterface
      *
      * @return void
      */
-    public function __construct($salt = '', $minHashLength = 0, $alphabet = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890')
+    public function __construct(
+        $salt = Hashids::DEFAULT_SALT,
```

Comment:
I think `self::DEFAULT_SALT` or `static::DEFAULT_SALT` is better.

What do you think?